test/output/callbacks/callbacks_test.py

```diff
@@ -65,6 +66,21 @@ def test_testrun(self, user_mock):
     testrun_output = BytesIO()
     mfg_inspector.OutputToTestRunProto(testrun_output)(record)
 
+    testrun = test_runs_pb2.TestRun()
+    testrun.ParseFromString(testrun_output.getvalue())
+    self.assertEqual(test_runs_pb2.FAIL, testrun.test_status)
```

Comment:
Hmmm... funny thing. In Python 2, test status is FAIL but in Python 3, it's ERROR. I guess this should be a bug, right? (In both Python 2 and 3 we expect to have the same test status.)

Reply:
What's the raised error in Python 3? It might be a 2 vs 3 incompatibility

Reply:
In Python 2, `testrun.test_status` is `FAIL`, but in Python 3, it's `ERROR`.

Reply:
I found why. In `examples.all_the_things.attachments`, the `test_attachment.data` is `bytes`-typed in Python 3, and thus `assert test_attachment.data == 'This is test attachment data.'` failed.